client/client_webhook.py

```diff
@@ -87,15 +87,14 @@ def process_webhook(self):
 
         # 3) Zip up the massaged files
         # context.aws_request_id is a unique ID for this lambda call, so using it to not conflict with other requests
-        zip_filename = commit_id + '.zip'
-        zip_filepath = os.path.join(self.base_temp_dir, zip_filename)
+        zip_filepath = tempfile.mktemp(dir=os.path.join(self.base_temp_dir))
```

Comment:
It looks like you forgot the file name. You are calling `os.path.join` with just 1 parameter.